code/modules/projectiles/projectile.dm

```diff
@@ -1143,11 +1143,16 @@
 	// Need to do this in order to prevent the ping from being deleted
 	addtimer(CALLBACK(I, TYPE_PROC_REF(/image, flick_overlay), src, 3), 1)
 
+/// Zonennote: debug attempt for bullet lag
```

Comment:
🚨 zonennote 🚨

Reply:
it worked so I'm keeping the change, removing the comment